JS_fibonacci.js

```diff
@@ -0,0 +1,5 @@
+function fibonacci(num) {
+  if (num <= 1) return 1;
+
+  return fibonacci(num - 1) + fibonacci(num - 2);
+}
```

Comment:
Hey, Appreciated Your Effort.  But this repo need a complete program not just the executing part.
Review Your Code Once.

Happy Coding!!

Reply:
The program needs to ask how many fibonacci terms should it needs to print.

Happy Coding!!